### DISCUSSION TITLE
attribute editing verification

### DISCUSSION THREAD

**Discussion by McKanty** [atrocore/atrocore#243] [Q&A] [answered]
Hi, I need to modify the attribute editing behavior: when the attribute type is set to array, I need to add verification that values are type one of specified in field tecdoc_data_type in attributes table. But I do not know where I should put that feature in what file?
I asked this question already in support, they told me that I need to create module for that. Any guides?

**Comment by atrolex** [comment, marked as answer]
You can use multienum type. In this case only preconfigured values can be selected. If it does not cover your need you need to programm custom module. The guide how to do it is available in our documentation, Currently there are no configurable validation rules for attributes. If you want us to develop a paid solution for your please write to support.
